<launch>

  <node name="tfPublisherClient" pkg="detection" type="tfPublisherClient" args="" respawn="false" output="screen">
  </node> 
  <node name="joy_node" pkg="joy" type="joy_node" args="" respawn="false" output="screen">
  </node>

</launch>